istioctl/cmd/workload.go

```diff
@@ -221,6 +220,17 @@ Configure requires either the WorkloadGroup artifact path or its location on the
 					return fmt.Errorf("workloadgroup %s not found in namespace %s: %v", name, namespace, err)
 				}
 			}
+
+			// extract the cluster ID from the injector config (.Values.global.multiCluster.clusterName)
+			if !validateFlagIsSetManuallyOrNot(cmd, "clusterID") {
+				fmt.Println("INFO: Flag clusterID is not set manually, extract from injector config")
+				// extract the cluster ID from the injector config if it is not set by user
+				clusterName := extractClusterIDFromInjectionConfig(kubeClient)
```

Comment:
This will not work if there is no Kubernetes Istio (e.g., in a pipeline, before Istio is installed). Can we fail in this case, instead of defaulting to `""`?

Reply:
yeah, I added one more return err to handler this case and there would be error return if there is no Istio installed.